### ISSUE TITLE
NullPointerException While updating Image/Text after layout Change.

### ISSUE THREAD

**Issue by ayushcshah** [smartdevicelink/sdl_java_suite#1465]
### Bug Report

SDL Application randomly crashes while I am trying to set Image/text after changing layout to `GRAPHIC_WITH_TEXT_AND_SOFTBUTTONS`. Same code was working before but just now I am getting following crash logs while trying to set image. The same is true for showing Text. If I dont change layout then app works as it is expected. I am using following code to change layout.

```java
 SetDisplayLayout setDisplayLayoutRequest = new SetDisplayLayout();
        setDisplayLayoutRequest.setDisplayLayout(PredefinedLayout.GRAPHIC_WITH_TEXT_AND_SOFTBUTTONS.toString());
        setDisplayLayoutRequest.setOnRPCResponseListener(new OnRPCResponseListener() {
            @Override
            public void onResponse(int correlationId, RPCResponse response) {
                if (((SetDisplayLayoutResponse) response).getSuccess()) {
                    Log.i("SdlService", "Display layout set successfully.");
                    updateSoftButton("Alert","Activity");
                    // Proceed with more user interface RPCs
                } else {
                    Log.i("SdlService", "Display layout request rejected.");
                }
            }

            @Override
            public void onError(int correlationId, Result resultCode, String info){
                Log.e(TAG, "onError: "+ resultCode+ " | Info: "+ info );
            }
        });
```

Crash Log.

```
2020-08-25 08:57:48.216 10832-10832/com.ford.research.BioSD E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.ford.research.BioSD, PID: 10832
    java.lang.NullPointerException: Attempt to invoke virtual method 'int java.lang.Enum.ordinal()' on a null object reference
        at com.smartdevicelink.managers.ManagerUtility$WindowCapabilityUtility.getMaxNumberOfMainFieldLines(ManagerUtility.java:77)
        at com.smartdevicelink.managers.screen.BaseTextAndGraphicManager.assembleShowText(BaseTextAndGraphicManager.java:412)
        at com.smartdevicelink.managers.screen.BaseTextAndGraphicManager.sdlUpdate(BaseTextAndGraphicManager.java:259)
        at com.smartdevicelink.managers.screen.BaseTextAndGraphicManager.update(BaseTextAndGraphicManager.java:219)
        at com.smartdevicelink.managers.screen.BaseScreenManager.commit(BaseScreenManager.java:579)
        at com.smartdevicelink.managers.screen.ScreenManager.commit(ScreenManager.java:45)
        at com.ford.research.BioSD.SdlService.isSafeToShowAlert(SdlService.java:621)
        at com.ford.research.BioSD.SdlService$10.run(SdlService.java:770)
        at android.os.Handler.handleCallback(Handler.java:751)
        at android.os.Handler.dispatchMessage(Handler.java:95)
        at android.os.Looper.loop(Looper.java:154)
        at android.app.ActivityThread.main(ActivityThread.java:6780)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1500)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1390)
```

##### Reproduction Steps

1.  Set layout to `GRAPHIC_WITH_TEXT_AND_SOFTBUTTONS`
2.  Change Text/Image

##### Expected Behavior

Update vehicle HMI with new Image or text

##### Observed Behavior

App crashes while trying to set image or text. After layout change.

##### OS & Version Information

*   Android Version: 7.0
*   SDL Android Version: 4.12.0
*   Testing Against: Ford SYNC 3.3 Build 18266

##### Test Case, Sample Code, and / or Example App

Change image.

```java
sdlManager.getScreenManager().beginTransaction();
sdlArtwork = new SdlArtwork(SDL_RED_IMAGE_FILENAME, FileType.GRAPHIC_JPEG, R.drawable.red, true);
sdlManager.getScreenManager().setPrimaryGraphic(sdlArtwork);
sdlManager.getScreenManager().commit(new CompletionListener() {
    @Override
    public void onComplete(boolean success) {
        Log.i(TAG, "ScreenManager update complete: " + success);
    }
});
```

Show Text

```java
sdlManager.getScreenManager().beginTransaction();
sdlManager.getScreenManager().setTextField1(text1);
sdlManager.getScreenManager().setTextField2(text2);
sdlManager.getScreenManager().setTextField3(text3);
sdlManager.getScreenManager().commit(new CompletionListener() {
    @Override
    public void onComplete(boolean success) {
        Log.i(TAG, "ScreenManager update complete: " + success);
    }
});
```

**Comment by mjuarez-ford** [comment]
Hello @joeygrover a partner is facing this issue on the library version 4.11.1 and updating to 5.0 could cause braking changes for them. Is it possible to merge this fix on the 4.11.1?

**Comment by mjuarez-ford** [comment]
Hello @bilal-alsharifi @joeygrover is there any update regarding the merging this issue with the 4.11.1?

**Comment by joeygrover** [comment]
As we are currently in the process of releasing 5.0.0 we can't guarantee a hotfix for the previous major version until after that release. We will also be applying the hotfix to the last minor release of the previous major version which will be 4.12.1. So the partner will need to update to that minor version.

**Comment by mjuarez-ford** [comment]
Thanks @joeygrover do you have a time frame for the release of 5.0 and the hotfix for the 4.12.1?

**Comment by joeygrover** [comment]
The 5.0.0 release is scheduled for 11/5. You can always find our planned releases as agreed to by the SDLC on the projects page https://github.com/smartdevicelink/sdl_java_suite/projects.

**Comment by theresalech** [comment]
Hi @mjuarez-ford, we have included the fix for this issue in 4.12.1, which is released here: https://github.com/smartdevicelink/sdl_java_suite/releases/tag/4.12.1.

It will also be included in the 5.0 release scheduled for 11/5. Thanks!

**Comment by mjuarez-ford** [comment]
Thanks!
